src/runtime/random.py

```diff
@@ -65,11 +66,13 @@ def expovariate(l: f64) -> f64:
     Return a random number from an exponential distribution with parameter
     `l` (lambda).
     """
+    assert l != 0.0
     return -_log(1.0 - random()) / l
```

Comment:
Instead of `assert` we can raise `ValueError` with some error message so that it is more clear.